### DISCUSSION TITLE
DatePicker 设置中文问题

### DISCUSSION THREAD

**Discussion by caoxuccc** [ant-design/ant-design#28280] [Q&A] [answered]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/antd-reproduction-template-forked-ctqbj?file=/index.js)

### Steps to reproduce
点击控件就能看到

### What is expected?
解决问题，找到原因

### What is actually happening?
配置国际化为中文应该都是中文

| Environment | Info |
|---|---|
| antd | 4.9.2 |
| React | 17.0.1 |
| System | macOS 11.0.1 |
| Browser | chorme  86.0.4240.198 |

**Comment by xrkffgg** [comment, marked as answer]
https://codesandbox.io/s/antd-reproduction-template-forked-y8l3h

```diff
+ import 'moment/locale/zh-cn';
```

**Comment by xrkffgg** [comment]
不太确定，能不能收到。 @一下好了

@greenPeppers
